pkg/client/group_mgmt_client.go

```diff
@@ -75,13 +75,14 @@ func (client *GroupMgmtClient) EnableDebug() {
 
 func (client *GroupMgmtClient) login(username, password string) (string, error) {
 	// Construct Payload
+	appName := "Go client"
```

Comment:
We need to append golang SDK version info as well as part of Go client login.

Reply:
Yes, I will fix it.